metro.config.js

```diff
@@ -5,15 +5,32 @@
 
 const {getDefaultConfig} = require('metro-config');
 const _ = require('underscore');
+require('dotenv').config();
 
 /* eslint arrow-body-style: 0 */
 module.exports = (() => {
+    const isUsingMockAPI = process.env.USE_MOCK_API === 'true' || process.env.E2E_TESTING === 'true';
```

Comment:
Is it really necessary to check both of these environment variables or can it just use one of them? It seems to me like USE_MOCK_API covers either case.

Reply:
USE_MOCK_API was just for testing, changed it to just use one env var 👍